### PR TITLE
[MB-8646] TOO is able to see crating details for MTO service items

### PR DESCRIPTION
**Pull request by hannaliebl**
## Description

Add the description to crating service items and change "Item Dimensions" and "Crate Dimensions" labels to "Item dimensions" and "Crate size".

## Reviewer Notes

Did I cover everything needed?

## Setup

PARAMS move code has the crating service items, go to the MTO page and you should see the new details.

```sh
make server_run
make client_run
```

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8646) for this change

## Screenshots
Screen Shot 2021-07-20 at 2 29 53 PM

### CODE REVIEW

**Comment by monfresh** [comment]
One thing I noticed is that the rejection modal in Zeplin doesn't show the reason (the reason for crating, as provided by the customer). @amanda5iu Can you confirm whether or not the customer reason should show up in the modal?

**Comment by monfresh** [comment]
Here are a couple more differences between this PR and Zeplin:

1. In Zeplin, the customer reason is not shown for the `uncrating` service item, but here we are showing it for all crating items.
2. When rejecting a crating service item, in Zeplin, there is a blank line between the Reason and the Rejection reason, but in this PR, there isn't a blank line.

**Comment by hannaliebl** [comment]
@monfresh thanks! I added space between the rejection reason and reason, and I've re-asked the two other questions in Slack.

File: src/components/Office/RequestedServiceItemsTable/RequestedServiceItemsTable.test.jsx [hunk 1]
```diff
@@ -46,24 +46,25 @@ const serviceItemWithDetails = {
...
+
+  expect(wrapper.find('.detailType').at(5).text()).toBe('Second Customer Contact:');
+  expect(wrapper.find('.detail dd').at(5).text().includes('2300Z')).toBe(true);
+  expect(wrapper.find('.detailType').at(6).text()).toBe('Second Available Delivery Date:');
+  expect(wrapper.find('.detail dd').at(6).text().includes('21 Sep 2020')).toBe(true);
+
+  expect(wrapper.find('.detailType').at(8).text()).toBe('ZIP:');
+  expect(wrapper.find('.detail dd').at(8).text().includes('20050')).toBe(true);
+  expect(wrapper.find('.detailType').at(7).text()).toBe('Reason:');
+  expect(wrapper.find('.detail dd').at(7).text().includes('Took a detour')).toBe(true);
```
Comment (duncan-truss) [inline src/components/Office/RequestedServiceItemsTable/RequestedServiceItemsTable.test.jsx:67]: Maybe we can cache some of these calls, I think each query selector will return a new array of nodes each time.
```javascript
const detailTypes = wrapper.find('.detailType')
const detailDefinitions = wrapper.find('.detail dd')

expect(detailTypes.at(0).text().toBe('Description');
expect(detailDefinitions.at(0).text().toBe('grandfather clock');
....
```

**Review by brooksr4 — APPROVED** [review]

**Pull request merged**